doc/man1/openssl-fipsinstall.pod.in

```diff
@@ -166,13 +166,7 @@ Configure the module to not perform run-time security checks as described above.
 
 =item B<-self_test_onload>
 
-Do not write the two fields related to the "test status indicator" and
-"MAC status indicator" to the output configuration file. Without these fields
-the self tests KATS will run each time the module is loaded. This option could be
-used for cross compiling, since the self tests need to run at least once on each
-target machine. Once the self tests have run on the target machine the user
-could possibly then add the 2 fields into the configuration using some other
-mechanism.
+This option is obsolete and does nothing.
```

Comment:
since FIPS 140-3 removed this capability.

Reply:
It's coming back based on comments from @t-j-h, so no need.